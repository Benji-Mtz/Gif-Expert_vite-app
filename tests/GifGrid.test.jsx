import { render } from '@testing-library/react';
import { GifItem } from '../src/components/GifItem';

describe('Testing a <GifItem />', () => {
  test('Se debe verificar el snapshot', () => {
    const { container } = render(
      <GifItem
        url={'google.com'}
        title={'Gouk'}
      />
    );

    expect(container).toMatchSnapshot();
  });
});
