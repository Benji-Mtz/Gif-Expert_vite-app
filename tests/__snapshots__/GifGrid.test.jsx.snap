// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing a <GifItem /> Se debe verificar el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Gouk"
      src="google.com"
    />
    <p>
      Gouk
    </p>
  </div>
</div>
`;
